libs/base/Control/Isomorphism.idr

```diff
@@ -81,14 +82,12 @@ eitherAssoc = MkIso eitherAssoc1 eitherAssoc2 ok1 ok2
 
 ||| Disjunction with false is a no-op
 eitherBotLeft : Iso (Either Void a) a
-eitherBotLeft = MkIso to from ok1 ok2
+eitherBotLeft = MkIso to from (\_ => Refl) ok2
   where to : Either Void a -> a
         to (Left x) = void x
         to (Right x) = x
         from : a -> Either Void a
         from = Right
-        ok1 : (x : a) -> to (from x) = x
```

Comment:
Leaving this as a separate function is more verbose, but I think the type signature helps you follow what's going on.

Reply:
Fair enough, changed back.